### ISSUE TITLE
Need a new websocket server code example

### ISSUE THREAD

**Issue by zydjohnHotmail** [kerryjiang/SuperSocket#479]
Hello:
I used your repo for WebSocket client before, it was a good one (version 1.6).
Now, I have to use WebSocket server, I want to use the 2.0 (even it is still in Beta).
But I can't find any good server example.
Can you provide an example, I found this issue: https://github.com/kerryjiang/SuperSocket/issues/447
It seems to be a rather good server example.
However, it contains other repos, which are not needed.
Could you please provide a server example like this one.
It will include WebSocketConfig, NewSessionConnected, NewMessageReceived, SessionClosed.
Do you think I can simply use his server code example, but I have to get rid of the following using:
using SuperWebSocket;
using SuperSocket.SocketBase.Config;
Thanks,

**Comment by kerryjiang** [comment]
These are samples:
https://github.com/kerryjiang/SuperSocket/tree/master/samples/WebSocketServer
https://github.com/kerryjiang/SuperSocket/tree/master/samples/WebSocketPushServer
https://github.com/kerryjiang/SuperSocket/tree/master/samples/LiveChat

**Comment by zydjohnHotmail** [comment]
Hello:
Thanks for your quick reply.  I need some advice for the WebSocketServer example.
For config:
{
    "serverOptions": {
        "name": "TestWebSocketServer",
        "listeners": [
            {
                "ip": "Any",
                "port": 4040
            },
            {
                "ip": "Any",
                "port": 4041,
                "security": "Tls12",
                "certificateOptions" : {
                    "filePath": "supersocket.pfx",
                    "password": "supersocket"
                }
            }
        ]
    }
}
Basically, I want to set up a simple WebSocketServer listen to localhost without SSL.  However, I want to set up different routes for the same port, like: localhost/login; localhost/upload.
I need to know how to change the config to do this.

**Comment by kerryjiang** [comment]
If you don't want to support TLS, just remove the tls listener node.
https://github.com/kerryjiang/SuperSocket/blob/master/samples/WebSocketPushServer/appsettings.json

In SuperSocket.WebSocket, we don't have a concept of routes. But you can get the path from the websocket session.

**Comment by zydjohnHotmail** [comment]
Hello:
How I can get the path from the websocket session? 
Do you have any code sample?

**Comment by kerryjiang** [comment]
https://github.com/kerryjiang/SuperSocket/blob/17b229dc9b8cf782f5327c9e210c48928d4dc306/test/SuperSocket.Tests/WebSocket/WebSocketBasicTest.cs#L117-L122
